### PR TITLE
Test for fastLeakTest

### PR DESCRIPTION
**Pull request by tpfau**
Adding Test for fastLeakTest and cleaning up the function. 

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [X] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: src/reconstruction/modelGeneration/fastLeakTest.m [hunk 1]
```diff
@@ -53,18 +50,24 @@
...
 end
 modelexchangesAbbr = unique([modelexchangesAbbr;rxnNames']);
-TestRxnNum = length(modelexchangesAbbr)
+TestRxnNum = length(modelexchangesAbbr);
 FluxExV =[];
 while cnt == 1
     modelClosed = changeObjective(modelClosed,modelexchangesAbbr);
     FF2=optimizeCbModel(modelClosed,'max');
-   ObjValue = FF2.f
+    if FF2.stat == 0
```
Comment (laurentheirendt) [inline src/reconstruction/modelGeneration/fastLeakTest.m]: Shouldn't this read: `if FF2.stat ~= 1` ?

Comment (tpfau) [reply src/reconstruction/modelGeneration/fastLeakTest.m]: No. The Error thrown here is only for an infeasible model. 
Admittedly, for other non optimal solutions there should probably also be an error, but they would mean very different things.

Comment (laurentheirendt) [reply src/reconstruction/modelGeneration/fastLeakTest.m]: The value of `FF2.f` when `FF2.stat == -1` or `FF2.stat == 2`  is `0`. Is that intended behaviour?

Comment (tpfau) [reply src/reconstruction/modelGeneration/fastLeakTest.m]: I'd say no. I added a error message for those instances.

Comment (laurentheirendt) [reply src/reconstruction/modelGeneration/fastLeakTest.m]: Thanks!

**Pull request merged**